src/UserStories/UserStories.css

```diff
@@ -157,18 +174,21 @@ html {
 .prev-button {
   grid-area: left;
   width: min-content;
-  margin-left: -50px;
+  /* this margin is for the button to be outside the story and it's -46 to match the real IG */
+  margin-left: -46px;
 }
 .next-button {
   grid-area: right;
   width: min-content;
   justify-self: flex-end;
+  /* this margin is for the button to be outside the story and it's -46 to match the real IG */
+  margin-right: -46px;
 }
 
 .story-footer {
   z-index: 2;
   position: relative;
-  margin-top: -184px;
+  margin-top: -41%;
```

Comment:
This broke the footer when you have a smaller screen.

Reply:
On it